core/store/store.go

```diff
@@ -76,4 +76,10 @@ type LedgerStore interface {
 	GetShardMsgsInBlock(blockHeight uint32, shardID common.ShardID) ([]xshard_types.CommonShardMsg, error)
 	GetRelatedShardIDsInBlock(blockHeight uint32) ([]common.ShardID, error)
 	GetShardMsgHash(shardID common.ShardID) (common.Uint256, error)
+	SaveCrossShardMsgByShardID(shardID common.ShardID, crossShardTxInfos []*types.CrossShardTxInfos) error
```

Comment:
suggest to change to multiple interfaces, rather a big one

Reply:
ok